Cython/Compiler/Nodes.py

```diff
@@ -4910,13 +4926,26 @@ def generate_execution_code(self, code):
         code.mark_pos(self.pos)
         code.pyclass_stack.append(self)
         cenv = self.scope
+        if self.orig_bases:
+            self.orig_bases.generate_evaluation_code(code)
         if self.bases:
             self.bases.generate_evaluation_code(code)
         if self.mkw:
             self.mkw.generate_evaluation_code(code)
         if self.metaclass:
             self.metaclass.generate_evaluation_code(code)
         self.dict.generate_evaluation_code(code)
+        if self.orig_bases:
+            # update __orig_bases__ if needed
+            code.putln("if (%s != %s) {" % (self.bases.result(), self.orig_bases.result()))
+            code.putln(
+                code.error_goto_if('PyDict_SetItemString(%s, "__orig_bases__", %s) == -1' % (
```

Comment:
The usual method in this case is `error_goto_if_neg()`.
```suggestion
                code.error_goto_if_neg('PyDict_SetItemString(%s, "__orig_bases__", %s)' % (
```